queries/v/highlights.scm

```diff
@@ -3,8 +3,17 @@
 
 ;; Identifiers -------------------
 (import_path) @namespace
-[(module_identifier) ] @variable.builtin
-[(identifier)] @variable
+(module_identifier) @variable.builtin
+(identifier) @variable
+(interpreted_string_literal) @string
+(string_interpolation) @none
+
+; TODO: Have differnt highlight to make then standout + highlight }{$ as special
+; ((string_interpolation
+;   (identifier) @constant
+;   "$" @punctuation.special
+;   "${" @punctuation.special
+;   "}" @punctuation.special))
```

Comment:
Okay it worked as you've suggested @theHamsta, however, couldn't quite figure out making identifiers highlight  differently

this is tree with two identifiers that start with `$`

```clojure
(interpreted_string_literal 
  (string_interpolation (identifier))
  (string_interpolation (identifier))
```

Reply:
I think this requires a change to the parser. Can you try to use a plain `"$"` here without the immediate https://github.com/vlang/vls/blob/e4ae12f117e7241a61db6a0c11e184d6ca0fe6b3/tree_sitter_v/grammar.js#L1585 or use ` alias(token.immediate("$"), "$")`. The braces seem to be handled by the external parser. In case you're using, nvim-treesitter/playground press `a` to see all anonymous tokens that are visible to the query engine.